typeDefs.js

```diff
@@ -273,6 +273,7 @@ const typeDefs = gql`
     "User Mutation"
     registerUser(user: userRegisterInput!): User
     loginUser(user: userLoginInput!): User
+    loginAdmin(user: UserInput!): User
```

Comment:
Please, rename the argument `user` to `loginInput` because it is data related to the login not the user itself. Also, please do it for the `loginUser` too

Reply:
Renamed!